### PR TITLE
2-stage Ignition: Install CoreOS to disk and provision an etcd cluster

### PR DESCRIPTION
**Pull request by dghubble**
hat tip @crawford

### CODE REVIEW

File: examples/ignition/coreos-install.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+      contents: |
+        [Unit]
+        Requires=network-online.target
+        After=network-online.target
+        [Service]
+        Type=oneshot
+        ExecStart=/usr/bin/coreos-install -d /dev/sda -C {{.coreos_channel}} -V {{.coreos_version}}
+        ExecStart=/usr/bin/udevadm settle
+        ExecStart=/usr/bin/mount /dev/disk/by-label/OEM /mnt
+        ExecStart=/bin/sh -c 'echo set linux_append=\\"coreos.config.url=\"{{.ignition_endpoint}}?{{.query}}&os=installed\" coreos.autologin\\" > /mnt/grub.cfg'
```
Comment (crawford) [inline examples/ignition/coreos-install.yaml]: You might want to leave off the `coreos.autologin`.

File: examples/ignition/coreos-install.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    - name: install.service
+      enable: true
+      contents: |
+        [Unit]
+        Requires=network-online.target
+        After=network-online.target
+        [Service]
+        Type=oneshot
+        ExecStart=/usr/bin/coreos-install -d /dev/sda -C {{.coreos_channel}} -V {{.coreos_version}}
+        ExecStart=/usr/bin/udevadm settle
```
Comment (crawford) [inline examples/ignition/coreos-install.yaml]: Ok, you can leave all of this off. `coreos-install` will support Ignition in the next OS release.

File: examples/ignition/coreos-install.yaml (hunk 1 above)
Comment (dghubble) [inline examples/ignition/coreos-install.yaml]: Removed.

File: examples/ignition/coreos-install.yaml (hunk 2 above)
Comment (dghubble) [inline examples/ignition/coreos-install.yaml]: :raised_hands: #46

**Pull request merged**